### PR TITLE
Add org.kde.ktimer

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://invent.kde.org/utilities/ktimer/-/issues/1
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://invent.kde.org/utilities/ktimer/-/merge_requests/3

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.kde.ktimer

**Comment by hfiguiere** [comment]
cc @flathub/kde

**Comment by barthalion** [comment]
Needs approval from KDE folks, but looks good to me at first glance.

File: org.kde.ktimer.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "ktimer",
+  "rename-icon": "ktimer",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11"
```
Comment (travier) [inline org.kde.ktimer.json:12]: Most likely missing `--device=dri`

File: org.kde.ktimer.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "app-id": "org.kde.ktimer",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "ktimer",
+  "rename-icon": "ktimer",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
```
Comment (travier) [inline org.kde.ktimer.json:10]: Most likely not needed.

**Comment by Mailaender** [comment]
I misunderstood the purpose until I installed it.

**Comment by Mailaender** [comment]
bot, build org.kde.ktimer

**Comment by Mailaender** [comment]
bot, build org.kde.ktimer

**Comment by hfiguiere** [comment]
> Note: I ramped up the permissions to IDE level, which might be controversial.

can you explain in detail why?

**Comment by barthalion** [comment]
> > Note: I ramped up the permissions to IDE level, which might be controversial.
> 
> can you explain in detail why?

The app is a task scheduler so users may expect they can run commands on the host.

**Comment by barthalion** [comment]
All good here @flathub/kde?

**Comment by travier** [comment]
This one needs testing to make sure it is useful/usable or if we need to make it beta until it's fixed.

**Pull request closed without merging**